### DISCUSSION TITLE
Ur5 with moveit

### DISCUSSION THREAD

**Discussion by nawrezz** [ros-industrial/universal_robot#666] [Q&A]
is it possible to get the same mouvement as movep with blend radius with moveit?

**Comment by NadiaHG** [comment]
@nawrezz could you please give more information? A more elaborated question will be very helpful 

Related post:https://answers.ros.org/question/380628/blend-radius-with-moveitrosur5e/

**Comment by fmauch** [comment]
I'll remove the wrid21 label as this is nothing that can be worked on directly. I'm not sure whether this can be resolved on this level, but to further clarify this, more input would be needed.
